python/Hash Table/hash_table_list.py

```diff
@@ -0,0 +1,54 @@
+class KeyValue:
+    def __init__(self, key, value):
+        self.key = key
+        self.value = value
+
+    def __repr__(self):
+        return f"Key: {self.key}\n Value: {self.value}"
```

Comment:
Line 7: Invalid Syntax,

Reply:
Here I am returning fstring, Python 3.6 and above supports this method.